oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/JwtIssuerValidator.java

```diff
@@ -32,20 +30,30 @@ public final class JwtIssuerValidator implements OAuth2TokenValidator<Jwt> {
 
 	private final JwtClaimValidator<Object> validator;
 
+	private boolean allowEmpty;
```

Comment:
Having thought about this further, I think we should wait on adding this toggle to validators that require the claim by default. We can relax it when a clear need arises.

Other than the error message cleanup you added in `validate`, let's please not make further functional changes to this class in this PR.